platformio.ini

```diff
@@ -18,7 +18,7 @@
 [platformio]
 src_dir      = Marlin
 boards_dir   = buildroot/share/PlatformIO/boards
-default_envs = mega2560
+default_envs = flsun_hispeedv1
```

Comment:
This should also be reverted to `mega2560`.

Reply:
Oups!!